### PR TITLE
redundant parentheses removed

### PR DESCRIPTION
**Pull request by EugeneDar**
Closes: https://github.com/objectionary/eo-strings/issues/109

### CODE REVIEW

**Comment by EugeneDar** [comment]
@volodya-lombrozo @yegor256 please have a look

File: src/main/eo/org/eolang/txt/text.eo [hunk 1]
```diff
@@ -99,9 +99,17 @@
       "".as-bytes
       [acc i x]
         if. > @
-          i.eq ((items.length).minus 1)
-          acc.concat (x.as-bytes)
-          (acc.concat (x.as-bytes)).concat s.as-bytes
+          eq.
```
Comment (volodya-lombrozo) [inline src/main/eo/org/eolang/txt/text.eo:102]: @EugeneDar To be honest, it looks less readable, as for me. 
What do you think about the next one?
```eo
        if. > @
          i.eq (items.length.minus 1)
          acc.concat x.as-bytes
          (acc.concat x.as-bytes).concat s.as-bytes
```

**Review by volodya-lombrozo — COMMENTED** [review]
@EugeneDar Please read the next [section](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword), please, and fix the PR description according with it.

File: src/main/eo/org/eolang/txt/text.eo (hunk 1 above)
Comment (EugeneDar) [reply src/main/eo/org/eolang/txt/text.eo:102]: @volodya-lombrozo sure, I'll rewrite it.

**Review by volodya-lombrozo — APPROVED** [review]
@EugeneDar Looks good to me

**Comment by EugeneDar** [comment]
@yegor256 please have a look

**Comment by yegor256** [comment]
@rultor merge

**Pull request merged**